doc/mgr/dashboard.rst

```diff
@@ -74,6 +95,9 @@ will be stored in the MON's configuration database::
 The password will be stored in the configuration database in encrypted form
 using ``bcrypt``. This is a global setting that applies to all dashboard instances.
 
+Accessing the dashboard
+^^^^^^^^^^^^^^^^^^^^^^^
+
 You can now access the dashboard using your (JavaScript-enabled) web browser, by
 pointing it to the selected TCP port and any of the host names or IP addresses
```

Comment:
Just found one minor room for improvement: could you put "TCP port" after "host names or IP addresses", so it reads as follows: "...pointing it to any of the host names or IP addresses and the selected TCP port..." (this would spare me from opening a separate PR in order to fix that). Thanks!

Reply:
@LenzGr OK, added a commit to address this.